mapbox_tilesets/utils.py

```diff
@@ -1,15 +1,27 @@
+import importlib
 import os
 import re
 
 import numpy as np
 
 from jsonschema import validate
 from requests import Session
-from supermercado.burntiles import burn
 
 import mapbox_tilesets
 
 
+def load_module(modulename):
+    """Dynamically imports a module and throws a readable exception if not found"""
+    try:
+        module = importlib.import_module(modulename)
+    except (ImportError):
+        raise ValueError(
```

Comment:
Happy for any suggestions here on getting a more readable or appropriate error back to console.

Reply:
I think it would help to add a link to the correct readme

Reply:
added e51d27